config/v1/types_feature.go

```diff
@@ -40,6 +40,9 @@ var (
 
 	// TopologyManager enables ToplogyManager support. Upgrades are enabled with this feature.
 	LatencySensitive FeatureSet = "LatencySensitive"
+
+	// InTreeVSphereVolumes disables CSI Migration for in-tree volumes on vSphere.
+	InTreeVSphereVolumes FeatureSet = "InTreeVSphereVolumes"
```

Comment:
If I understand correctly, in 4.12 this would be harmless and do nothing, so a `""`->`InTreeVSphereVolumes`->`""` transition is ok, in 4.13, we don't have the CCM tested for uninstalling (eg CCM -> KCM) migration, as this was something we never intended to support, so should we, in 4.13, prevent the transition `""`->`InTreeVSphereVolumes`, as this would be an unsupported scenario?

CC @deads2k IIUC this should be ok as a CEL validation, can you think of some reason introducing a transition prevention would be a breaking change here?